homestar-invocation/src/pointer.rs

```diff
@@ -278,6 +346,34 @@ where
     }
 }
 
+impl JsonSchema for Pointer {
+    fn schema_name() -> String {
+        "pointer".to_owned()
```

Comment:
can we not automatically derive this?

Reply:
As discussed, we can't derive schemars when we implement JsonSchema, and if we could, we would still need to rename this from title case to lowercase.